Commands.adoc

```diff
@@ -0,0 +1,244 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you `vagrant up` and have a running VM, the vagrant-service-manager
+ enables you to check and tweak your container development environment.
```

Comment:
How about:

Once you start the ADB virtual machine using `vagrant up`, you can use `vagrant service-manager` to manage your development environment.